transformation/src/test/test-files/inputs/File/part-1.json

```diff
@@ -1,4 +1,9 @@
level":50,"path":"/experiments/ENCSR467QPI/","name":"audit_experiment","detail":"Fastq file {ENCFF120BBO|/files/ENCFF120BBO/} has read length of 86bp, while the recommended read length for human data is > 100bp.","category":"insufficient read length"},{"level_name":"NOT_COMPLIANT","level":50,"path":"/experiments/ENCSR467QPI/","name":"audit_experiment","detail":"Fastq file {ENCFF765PVL|/files/ENCFF765PVL/} has read length of 87bp, while the recommended read length for human data is > 100bp.","category":"insufficient read length"}],"ERROR":[{"level_name":"ERROR","level":60,"path":"/experiments/ENCSR467QPI/","name":"audit_experiment","detail":"Replicate of experiment processed by WGBS single-end pipeline ( {ENCPL936TND|/pipelines/ENCPL936TND/} ) has a coverage of 0X. The minimum ENCODE standard coverage for each replicate in a WGBS assay is 25X and the recommended value is > 30X (See {ENCODE WGBS data standards|/data-standards/wgbs/} )","category":"extremely low coverage"},{"level_name":"ERROR","level":60,"path":"/experiments/ENCSR467QPI/","name":"audit_experiment","detail":"Replicate of experiment processed by WGBS single-end pipeline ( {ENCPL936TND|/pipelines/ENCPL936TND/} ) has a coverage of 0X. The minimum ENCODE standard coverage for each replicate in a WGBS assay is 25X and the recommended value is > 30X (See {ENCODE WGBS data standards|/data-standards/wgbs/} )","category":"extremely low coverage"}]},"award":"/awards/ROADMAP/","biological_replicates":[2],"biosample_ontology":"/biosample-types/cell_line_EFO_0001196/","cloud_metadata":{"file_size":1545668172,"url":"https://encode-public.s3.amazonaws.com/2016/10/11/87818cbf-d9b8-4afe-9ddf-14dde0cafa15/ENCFF797REO.fastq.gz","md5sum_base64":"NFMV3nBzwD0JYo4VGl5Zuw=="},"content_md5sum":"243dcf14a07425a7766577ab6a76d7bd","dataset":"/experiments/ENCSR467QPI/","date_created":"2016-10-11T21:05:58.194169+00:00","dbxrefs":["SRA:SRR020257"],"fastq_signature":["TEMP:3:1::@EVANS_3021:3:1:0:1061"],"file_format":"fastq","file_size":1545668172,"file_type":"fastq","flowcell_details":[],"href":"/files/ENCFF797REO/@@download/ENCFF797REO.fastq.gz","lab":"/labs/zhiping-weng/","library":"/libraries/ENCLB258GAJ/","md5sum":"345315de7073c03d09628e151a5e59bb","no_file_available":false,"output_category":"raw data","output_type":"reads","platform":"/platforms/OBI%3A0000703/","quality_metrics":[],"read_count":20481685,"read_length":87,"read_length_units":"nt","replicate":"/replicates/87dae9d3-f68e-4b26-a2b7-a3096f2e2c87/","replicate_libraries":["/libraries/ENCLB258GAJ/"],"run_type":"single-ended","s3_uri":"s3://encode-public/2016/10/11/87818cbf-d9b8-4afe-9ddf-14dde0cafa15/ENCFF797REO.fastq.gz","schema_version":"17","status":"released","submitted_by":"/users/5a1c0169-f9d0-4598-8f92-3eb84b122c1e/","submitted_file_name":"SRR020257.fastq.gz","superseded_by":[],"technical_replicates":["2_2"],"title":"ENCFF797REO","uuid":"87818cbf-d9b8-4afe-9ddf-14dde0cafa15","read_structure":[{"sequence_element":"fake","start":0,"end":1000000}]}
```

Comment:
Note: I couldn't actually find any files with a defined `read_structure` in our extracted data, so I added a fake one based on ENCODE's published schema to this line.